app/connectors/cache/SessionDataCacheConnector.scala

```diff
@@ -32,18 +32,17 @@ import javax.inject.Inject
 class SessionDataCacheConnectorImpl @Inject()(config: FrontendAppConfig, http: HttpClientV2)
     extends SessionDataCacheConnector {
 
-  private def url(cacheId: String): String =
-    s"${config.pensionsAdministrator}/pension-administrator/journey-cache/session-data/$cacheId"
+  private def url: String = s"${config.pensionsAdministrator}/pension-administrator/journey-cache/session-data-self"
 
   override def fetch(cacheId: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[SessionData]] =
     http
```

Comment:
Do you need to pass in some header? `loggedInAsPsa` maybe?

Reply:
Nope. I looked Pension Administrator, now they obtain the cacheId(externalId) in a same way as we used to (extract it from auth). Therefore, we don't need to send them anything.